pkg/claims/run.go

```diff
@@ -99,20 +103,40 @@ func (r Run) ShouldRecord() bool {
 // ToCNAB associated with the Run.
 func (r Run) ToCNAB() cnab.Claim {
 	return cnab.Claim{
-		SchemaVersion: CNABSchemaVersion(),
-		ID:            r.ID,
 		// CNAB doesn't have the concept of namespace, so we smoosh them together to make a unique name
+		SchemaVersion:   CNABSchemaVersion(),
+		ID:              r.ID,
 		Installation:    r.Namespace + "/" + r.Installation,
 		Revision:        r.Revision,
 		Created:         r.Created,
 		Action:          r.Action,
 		Bundle:          r.Bundle,
 		BundleReference: r.BundleReference,
-		Parameters:      r.Parameters,
+		Parameters:      r.TypedParameterValues(),
 		Custom:          r.Custom,
 	}
 }
 
+// TypedParameterValues returns parameters values that have been converted to
+// its typed value based on its bundle definition.
+func (r Run) TypedParameterValues() map[string]interface{} {
```

Comment:
Can you add a unit test in run_test.go for this function?

Reply:
added!